ibis/backends/clickhouse/tests/test_select.py

```diff
@@ -380,7 +380,9 @@ def test_asof_join(time_left, time_right):
     expected = pd.merge_asof(
         time_left.execute(), time_right.execute(), on="time", suffixes=("", "_right")
     )
-    tm.assert_frame_equal(result[expected.columns], expected)
+    tm.assert_frame_equal(
+        result[expected.columns].fillna(pd.NA), expected.fillna(pd.NA)
+    )
```

Comment:
Do you remember what was the error here?

Reply:
Ibis fails tests on warnings we don't explicitly allow and this is warning about eventually not comparing nans and NAs

Reply:
This is so that our end users are not blasted with gobs of spam every time a new version of a dependency is released.